controller/clusters_blackbox_test.go

```diff
@@ -300,6 +300,41 @@ func (s *ClustersControllerTestSuite) TestCreate() {
 	})
 }
 
+func (s *ClustersControllerTestSuite) TestDelete() {
+
+	s.T().Run("ok", func(t *testing.T) {
+		// given
+		sa := &authtestsupport.Identity{
+			Username: authsupport.ToolChainOperator,
+			ID:       uuid.NewV4(),
+		}
+		c := testsupport.CreateCluster(t, s.DB)
+		svc, ctrl := s.newSecuredControllerWithServiceAccount(sa)
+		// when/then
+		test.DeleteClustersNoContent(t, svc.Context, svc, ctrl, c.ClusterID)
```

Comment:
Also check that the cluster was actually deleted from DB.

Reply:
done in 870041f